cli/cli/commands/engine/restart/restart.go

```diff
@@ -92,6 +94,13 @@ var RestartCmd = &lowlevel.LowlevelKurtosisCommand{
 			Type:      flags.FlagType_String,
 			Default:   defaultDomain,
 		},
+		{
+			Key:       logRetentionPeriodFlagKey,
```

Comment:
is `1w` `1d` 1m` valid; can we add more examples here!

Reply:
Makes sense! for context I'm going off whatever `time.ParseDuration` supports https://pkg.go.dev/time#ParseDuration but in the future can add custom logic to support other time periods or use a another library